irohad/ametsuchi/impl/postgres_block_index.cpp

```diff
@@ -27,139 +33,54 @@ namespace {
         },
         [](const auto &) -> ReturnType { return boost::none; });
   }
+}  // namespace
 
-  // make index tx hash -> block where hash is stored
-  std::string makeHashIndex(
-      const shared_model::interface::types::HashType &hash,
-      shared_model::interface::types::HeightType height,
-      size_t index) {
-    boost::format base(
-        "INSERT INTO position_by_hash(hash, height, index) VALUES ('%s', "
-        "'%s', '%s');");
-    return (base % hash.hex() % height % index).str();
-  }
-
-  std::string makeCommittedTxHashIndex(
-      const shared_model::interface::types::HashType &rejected_tx_hash) {
-    boost::format base(
-        "INSERT INTO tx_status_by_hash(hash, status) VALUES ('%s', "
-        "TRUE);");
-    return (base % rejected_tx_hash.hex()).str();
+// Collect all assets belonging to creator, sender, and receiver
```

Comment:
Maybe this docstring can be moved to hpp.

Reply:
it is a private method, so it's doc is more useable in cpp